src/pages/ReimbursementAccount/BankInfo/BankInfo.tsx

```diff
@@ -1,56 +1,51 @@
 import {View} from 'react-native';
 import {withOnyx} from 'react-native-onyx';
+import {OnyxEntry} from 'react-native-onyx/lib/types';
 import HeaderWithBackButton from '@components/HeaderWithBackButton';
 import InteractiveStepSubHeader from '@components/InteractiveStepSubHeader';
 import ScreenWrapper from '@components/ScreenWrapper';
 import useLocalize from '@hooks/useLocalize';
 import useSubStep from '@hooks/useSubStep';
+import {SubStepProps} from '@hooks/useSubStep/types';
 import useThemeStyles from '@hooks/useThemeStyles';
 import getPlaidOAuthReceivedRedirectURI from '@libs/getPlaidOAuthReceivedRedirectURI';
-import reimbursementAccountDraftPropTypes from '@pages/ReimbursementAccount/ReimbursementAccountDraftPropTypes';
-import {reimbursementAccountPropTypes} from '@pages/ReimbursementAccount/reimbursementAccountPropTypes';
 import * as ReimbursementAccountProps from '@pages/ReimbursementAccount/reimbursementAccountPropTypes';
-import getDefaultValueForReimbursementAccountField from '@pages/ReimbursementAccount/utils/getDefaultValueForReimbursementAccountField';
 import getSubstepValues from '@pages/ReimbursementAccount/utils/getSubstepValues';
 import * as BankAccounts from '@userActions/BankAccounts';
 import CONST from '@src/CONST';
 import ONYXKEYS from '@src/ONYXKEYS';
+import type {ReimbursementAccount, ReimbursementAccountDraft} from '@src/types/onyx';
 import Confirmation from './substeps/Confirmation';
 import Manual from './substeps/Manual';
 import Plaid from './substeps/Plaid';
 
-const propTypes = {
+type BankInfoOnyxProps = {
     /** Plaid SDK token to use to initialize the widget */
-    plaidLinkToken: PropTypes.string,
+    plaidLinkToken: OnyxEntry<string>;
 
     /** Reimbursement account from ONYX */
-    reimbursementAccount: reimbursementAccountPropTypes,
+    reimbursementAccount: OnyxEntry<ReimbursementAccount>;
 
     /** The draft values of the bank account being setup */
-    reimbursementAccountDraft: reimbursementAccountDraftPropTypes,
+    reimbursementAccountDraft: OnyxEntry<ReimbursementAccountDraft>;
 };
 
-const defaultProps = {
-    plaidLinkToken: '',
-    reimbursementAccount: ReimbursementAccountProps.reimbursementAccountDefaultProps,
-    reimbursementAccountDraft: {},
-};
+type BankInfoProps = BankInfoOnyxProps;
 
 const bankInfoStepKeys = CONST.BANK_ACCOUNT.BANK_INFO_STEP.INPUT_KEY;
-const manualSubsteps = [Manual, Confirmation];
-const plaidSubsteps = [Plaid, Confirmation];
+const manualSubsteps: Array<React.ComponentType<SubStepProps>> = [Manual, Confirmation];
+const plaidSubsteps: Array<React.ComponentType<SubStepProps>> = [Plaid, Confirmation];
 const receivedRedirectURI = getPlaidOAuthReceivedRedirectURI();
 
-function BankInfo({reimbursementAccount, reimbursementAccountDraft, plaidLinkToken}) {
+function BankInfo({reimbursementAccount = ReimbursementAccountProps.reimbursementAccountDefaultProps, reimbursementAccountDraft = {}, plaidLinkToken = ''}: BankInfoProps) {
```

Comment:
Do we need `reimbursementAccount` and `reimbursementAccountDraft`? I guess optional chaining should handle it, could you confirm?

Reply:
Yes, both are needed, we first try to get values for the for from draft (user recently filled them in) and then from regular reimbursementAccount (values already saved in BE)

Reply:
But looking at the other comment my guess is that you wanted to ask if we need default values for RA and RAD?

Reply:
Then actually yes, these are not needed

Reply:
After thinking it through I dont think we need default value for `reimbursementAccountDraft` but its needed for `reimbursementAccount`

Reply:
Yes I think we need to remove only the default value of `reimbursementAccountDraft`

Reply:
@MrMuzyk Have you checked if `reimbursementAccount` default params are ever used? I guess if Onyx will pass `null` it will use `null` instead of defaults

Reply:
Not yet, still doing remaining changes. But I think that @VickyStash is right. When Onyx will return null because it wont be populated yet it wont use the default prop anyway as react only uses it when undefined is passed

Reply:
I wonder if reimbursementAccountDefaultProps should be moved somewhere else, because *proptypes file will be removed in the future.

Reply:
Yeah, it's working fine without default value. It was being completely ignored anyway